modules/wsl-distro.nix

```diff
@@ -9,6 +9,7 @@ in
 
   options.wsl = with types; {
     enable = mkEnableOption "support for running NixOS as a WSL distribution";
+    useWindowsDriver = mkEnableOption "Use windows host as OpenGL driver";
```

Comment:
`mkEnableOption` will automatically prepend "Enable" to the start of its description. Also, Windows, with a capital.

Reply:
Thank you! Updated